.github/workflows/publish.yaml

```diff
@@ -31,6 +31,7 @@ on:
       - main
 
 permissions:
```

Comment:
I'd put thus under the job itself, right before `steps`.

Reply:
done, thank you